erizo/src/erizo/media/ExternalOutput.cpp

```diff
@@ -376,8 +386,9 @@ bool ExternalOutput::initContext() {
     video_stream_->codec->width = 640;
     video_stream_->codec->height = 480;
     video_stream_->time_base = (AVRational) { 1, 30 };
+    video_stream_->metadata = genVideoMetadata();
```

Comment:
hmmm... this solution will only work if the video is always in the same orientation, am I right?